### ISSUE TITLE
Missing parameters

### ISSUE THREAD

**Issue by ghost** [ToddGreenfield/homebridge-nut#5]
Hi,

thank you so much for this great plugin, it should be very useful.

I configured my config.json with this:

```
...
{
         "platform": "Nut",
         "name": "Nut",
         "host": "localhost",
         "port": "3493",
         "search_time_delay": "1",
         "low_batt_warning": "40",
         "polling": "120"
}
... 
```

but when I start home bridge I found 

```
[8/14/2017, 11:17:04 AM] Loading 0 accessories...
Scan this code with your HomeKit App on your iOS device to pair with Homebridge:
                        
      
                        
[8/14/2017, 11:17:06 AM] Missing parameters.
[8/14/2017, 11:17:07 AM] Homebridge is running on port 51826.
```

Why "Missing parameters" ? In fact home bridge is not showing any new ups device in homelike.

Thank you

**Comment by ToddGreenfield** [comment]
Are you able to successfully run other plugins?
The error doesn't look like one this plugin would throw.

**Comment by ToddGreenfield** [comment]
Have you been able to get this working? Are you able to run another plugin like homebridge-dummy?

**Comment by ghost** [comment]
Yes, other plugins are working fine

**Comment by ToddGreenfield** [comment]
Are you still having this issue? I pushed a new update, please update and try that. Also, are you able to successfully run the test.js script locally and see the output from your running NUT server? Please attach the output from that script.

**Comment by ToddGreenfield** [comment]
@CryogenicPilot please also show what your ups.conf configuration looks like in the section where your ups is defined.

**Comment by ToddGreenfield** [comment]
@CryogenicPilot would you please run this command in your NUT server console, or change "localhost" with your server IP and/or 3493 with your NUT server port:

echo "LIST UPS" | nc localhost 3493

And post the response here please. This may be related to #7 which we may have a fix for shortly.

**Comment by ToddGreenfield** [comment]
This should be resolved with the 1.0.11 update.
